tests/integration/p2p.go

```diff
@@ -74,10 +76,10 @@ type SubscribeToCollection struct {
 	// them to this node.
 	NodeID int
 
-	// CollectionIDs are the collection IDs (indexes) of the collections to subscribe to.
+	// CollectionID is the collection ID (indexes) of the collection to subscribe to.
 	//
 	// A [NonExistentCollectionID] may be provided to test non-existent collection IDs.
-	CollectionIDs []int
+	CollectionID int
```

Comment:
praise: This is a good change, thanks - the test action should reflect the structure of the production operation it represents.  `AddP2PCollection` takes a single id, so so should the action IMO

Reply:
I'm happy to consider either option as long as it matches the client API.

Reply:
I'll let you decide which one you think creates a better user experience. 

```
defradb client p2pcollection add Users
defradb client p2pcollection add Address
```
vs
```
defradb client p2pcollection add Users,Address
```

Reply:
I've updated it to support multiple collections like we discussed.